lib/orocos/process.rb

```diff
@@ -1162,7 +1172,9 @@ def self.resolve_all_tasks(process, cache = Hash.new)
 
         # Wait for a process to become reachable
         #
-        def self.wait_running(process, timeout = nil, name_service = Orocos::CORBA.name_service, &block)
+        def self.wait_running(
+            process, timeout = nil, ior_read = nil, ior_message = "", &block
+        )
             if timeout == 0
```

Comment:
While you are at it, please refactor this method into two ... one that is the "zero-timeout case", could be `resolve_running_tasks` or `test_if_deployment_is_running` or something. The rest of the logic would stay here.